projects/ad9695_fmc/README.md

```diff
@@ -0,0 +1,38 @@
+
+# AD9695-EVAL HDL reference design
+
+## Overview
+
+The following design supports Xilinx FPGA's. The AD9695 EVAL prototyping system connects to the FPGA carrier board through a FMC (FPGA Mezzanine Cad) high pin count connector.  
+
+Currently supported carriers:
+
+|  Carrier name | FMC connector |
+| ------------- | ------------- |
+|  ZCU102       |   HPC1        |
+
+## Architecture
+
+The main scope of the HDL design is to provide all the required digital interfaces
+for the data acquisition board of the prototyping system.
+
+The following block diagram presents the simplified system architecture: 
+
+![HDL Block Diagram](./doc/img/ad9695_fmc_01.svg)
+
+## Clocking
+
+![Clocking](./doc/img/ad9695_fmc_clocking_01.svg)
+
```

Comment:
We should keep the format of the Readme files similar with the other projects for now.